contracts/core/tokens/RebalancingSetToken.sol

```diff
@@ -341,10 +341,21 @@ contract RebalancingSetToken is
     function endFailedAuction()
         external
     {
+        (
+            ,
+            uint256 calculatedUnitShares
+        ) = StandardSettleRebalanceLibrary.calculateNextSetIssueQuantity(
```

Comment:
Can we write another function in `StandardSettleRebalanceLibrary` called something like `canPeformSuccessfulSettlement` or so instead of passing through calculatedUnitShares?